### PR TITLE
CWS internationalize refactor

### PR DESCRIPTION
**Pull request by AmySteam**
Fixes #SOME_ISSUE_NUMBER

Changes proposed in this pull request:

- Move Store Listing localization information under CWS Dashboard "Store Listing" tab
- Move locales list to extensions i18n
- Move list of regions to the CWS Dashboard "Distribution" tab
- Remove CWS internationalize

### CODE REVIEW

**Comment by AmySteam** [comment]
@jpmedley This is ready for a "light" review. I changed screenshots and content to fit the localization information. Please check the description to know what changes were necessary since we are removing [CWS internationalize](https://developer.chrome.com/docs/webstore/i18n/) . This decision was approved by product.

File: site/en/docs/webstore/cws-dashboard-listing/index.md [hunk 1]
```diff
@@ -6,62 +6,103 @@ updated: 2023-05-01
 description: How to add listing information for your Chrome Web Store item.
 ---
 
-This page describes the fields you must fill out to complete your store listing. To learn how to
-make your listing more compelling, be sure to read [Creating a great listing page][best-listing].
+This page describes how to fill out the **Store listing** tab in the developer dashboard. This is
```
Comment (jpmedley) [inline site/en/docs/webstore/cws-dashboard-listing/index.md:9]:
```suggestion
This page describes how to fill out the **Store listing** tab in the Developer Dashboard. This is
```

File: site/en/docs/webstore/cws-dashboard-listing/index.md [hunk 2]
```diff
@@ -6,62 +6,103 @@ updated: 2023-05-01
...
-  
-{% Aside 'caution' %}
-Be detailed but make sure your description complies with the [Keyword Spam][keyword-spam] policy.
-{% endAside %}
-
-- The **primary [category][categories]** where your item should be listed.
-- Your item's **language** to help users find it.
+- A **detailed description** of your item. This helps users understand why they should install your
+  extension. Start your description with a concise statement of what your item does, so
+  users can understand the main idea and features at a glance. After that provide additional details, promotional copy, update logs, and so forth. Be detailed but ensure it complies with the [Keyword Spam][keyword-spam] policy.
```
Comment (jpmedley) [inline site/en/docs/webstore/cws-dashboard-listing/index.md:25]:
```suggestion
  users can understand the main idea and features at a glance. After that provide additional details, promotional copy, update logs, and so forth. Be detailed but ensure it complies with the [Keyword spam][keyword-spam] policy.
```

File: site/en/docs/webstore/cws-dashboard-listing/index.md [hunk 3]
```diff
@@ -6,62 +6,103 @@ updated: 2023-05-01
...
+{% Img src="image/BhuKGJaIeLNPW9ehns59NfwqKxF2/70mdctbPy8hIIHPFxvZF.png",
+alt="Detailed description of the Store Listing", width="483", height="140" %}
+
+If you support multiple locales, repeat the previous two steps until you've provided a detailed
+description for each locale.
+
+### Localize screenshots and promotional video
+
+You can also tailor screenshots and promotional video for each locale. Once you've choose the
+locale. add a screenshot by dropping your screenshots in the **Localized screenshots** section. If
```
Comment (jpmedley) [inline site/en/docs/webstore/cws-dashboard-listing/index.md:80]:
```suggestion
locale, add a screenshot by dropping your screenshots in the **Localized screenshots** section. If
```

**Review by jpmedley — APPROVED** [review]
I went ahead and approved. Please merge the 3 nits and ping me on chat if that prevents you from merging.

**Pull request merged**